src/main/java/com/gooddata/project/ProjectValidationResultGdcTimeElParam.java

```diff
@@ -6,31 +6,52 @@
 import com.fasterxml.jackson.annotation.JsonTypeName;
 
 import java.util.List;
+import java.util.Objects;
 
 @JsonTypeName("gdctime_el")
 @JsonIgnoreProperties(ignoreUnknown = true)
-public class ProjectValidationResultGdcTimeElParam extends ProjectValidationResultElParam {
+public class ProjectValidationResultGdcTimeElParam extends ProjectValidationResultParam {
 
-    ProjectValidationResultGdcTimeElParam(final List<String> ids, final List<String> vals) {
-        super(ids, vals);
+    private final List<String> ids;
+
+    ProjectValidationResultGdcTimeElParam(final List<String> ids) {
+        this.ids = ids;
     }
 
     @JsonCreator
-    private static ProjectValidationResultGdcTimeElParam create(@JsonProperty("ids") List<String> ids, @JsonProperty("vals") List<String> vals) {
-        return new ProjectValidationResultGdcTimeElParam(ids, vals);
+    private static ProjectValidationResultGdcTimeElParam create(@JsonProperty("ids") List<String> ids) {
+        return new ProjectValidationResultGdcTimeElParam(ids);
     }
 
-    @Override
-    public boolean equals(final Object o) {
-        if (this == o) return true;
-        if (!(o instanceof ProjectValidationResultGdcTimeElParam)) return false;
+    public List<String> getIds() {
+        return ids;
+    }
 
-        ProjectValidationResultGdcTimeElParam that = (ProjectValidationResultGdcTimeElParam) o;
+    /**
+     * @deprecated for backward compatibility only. Do not use this method, it always returns null.
+     */
+    @Deprecated
+    public List<String> getVals() {
+        return null;
```

Comment:
The method returned `null` value before the fix (JSON structure doesn't contain `vals` node) so I would prefer to preserve this as it is for ensuring previous behaviour.